api/admin.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015 Spinpunch, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package api
+
+import (
+	"bufio"
+	"net/http"
+	"os"
+
+	l4g "code.google.com/p/log4go"
+	"github.com/gorilla/mux"
+
+	"github.com/mattermost/platform/model"
+	"github.com/mattermost/platform/utils"
+)
+
+func InitAdmin(r *mux.Router) {
+	l4g.Debug("Initializing admin api routes")
+
+	sr := r.PathPrefix("/admin").Subrouter()
+	sr.Handle("/logs", ApiUserRequired(getLogs)).Methods("GET")
+}
+
+func getLogs(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Do we have a limit on log file size? If not this could get pretty large. Limit? Paging? 0/5 on the solution.